docs/platform/webcontainers/project-config.md

```diff
@@ -125,7 +125,7 @@ A map of default environment variables that will be set in each top-level shell
 
 Use the `terminal` parameter in the project’s URL query string to select one or several scripts from `package.json` to run on project load.
 
-This can be useful when a project doesn’t define a custom `startCommand`, or if you want to share a link that runs a different script then the one specified in `startCommand`.
```

Comment:
the first "that" is correct this is not correct

Reply:
Could you elaborate, please? I think I'm a little lost.

Reply:
Ohhhh right.

Reply:
Would you like to suggest a correction? 
Please note that it should be "than" (a comparison) and not "then" (which means 'after').
You could push a new commit replacing:
- the "than" with "that" 
- the "then" with "than" :)

So:

This can be useful when a project doesn’t define a custom `startCommand`, or if you want to share a link that runs a different script than the one specified in `startCommand`.